src/amo/api/abuse.js

```diff
@@ -25,12 +25,12 @@ export type AbuseReporter = {|
 export type ReportAddonParams = {|
   addonId: string,
   api: ApiState,
-  reporterName: string | null,
-  reporterEmail: string | null,
+  reporter_name: string | null,
+  reporter_email: string | null,
   message: string | null,
   reason: string | null,
   location: string | null,
-  addonVersion: string | null,
+  addon_version: string | null,
```

Comment:
Since it somewhat goes against the usual naming conventions for addons-frontend could you add a comment so nobody is tempted to rename this in the future only to realize it makes things more annoying ?